nixos/user.nix

```diff
@@ -0,0 +1,5 @@
+{ 
+  name = "carlos"; 
```

Comment:
how are we going to deal with this file? If I pull these changes it will fail for my current setup. should we ignore this file and add a template to the documentation?